plugins/wazo-gigaset/Nx70-83.v2.49.1/plugin-info

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.1.0",
+    "description": "Plugin for Gigaset N670 IP PRO and N870 IP PRO.",
+    "description_fr": "Greffon pour Gigaset N670 IP PRO et N870 IP PRO.",
+    "capabilities": {
+        "Gigaset, N670 IP PRO, 83.V2.49.1": {
+            "lines": 20,
+            "ha": true,
+            "expansions": false,
+            "switchboard": false,
+            "proto": "sip"
```

Comment:
Could you please rename the keys to the ones [defined in the README](https://github.com/wazo-platform/wazo-provd-plugins#plugin-information-files) please? I know this is how it is in the current available plugins (not your fault then) but we need to get rid of those old keys :smile: 
Thank you!